cmd/root.go

```diff
@@ -124,7 +129,7 @@ func init() {
 	// will be global for your application.
 	// rootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.odo.yaml)")
 
-	rootCmd.PersistentFlags().BoolVarP(&GlobalVerbose, "verbose", "v", false, "Verbose output")
+	rootCmd.PersistentFlags().StringVarP(&GlobalVerbose, "verbose", "v", "", "Verbose output")
```

Comment:
description: "log level for V logs"
default value: 0

Reply:
can we have `IntVarP` here, instead of taking string input ?

Reply:
Need it as a string as in line 105.

But yes, I think making it int and using IntVarP would provide additional advantage of additional pre-validating it as an int and anyway int to string is anyway doable at least additional cost :)

Reply:
@cdrage No the glog uses integer verbosity levels to indicate different levels of verbosity of info